### PR TITLE
driver/power: add PDUDaemon support

### PR DESCRIPTION
**Pull request by jluebbe**
**Description**
This addresses the long-standing request to add support for PDUDaemon
ports via the PowerProtocol.

**Checklist**
- [x] Documentation for the feature
- [ ] Tests for the feature 
- [x] The arguments and description in doc/configuration.rst have been updated
- [ ] Add a section on how to use the feature to doc/usage.rst
- [ ] CHANGES.rst has been updated
- [x] PR has been tested

### CODE REVIEW

File: labgrid/driver/powerdriver.py [hunk 1]
```diff
@@ -279,3 +279,49 @@ def get(self):
...
+            res += "&delay={}".format(self.delay)
+        return res
+
+    def on_activate(self):
+        self._host, self._port = proxymanager.get_host_and_port(self.port, default_port=16421)
+
+    @Driver.check_active
+    @step()
+    def on(self):
+        self._requests.get(self._build_url('on'))
```
Comment (Emantor) [inline labgrid/driver/powerdriver.py]: According to the README, pdudaemon can return 503 as a status if the request is not accepted, we should raise an error/exception if this is the case.

Comment (jluebbe) [reply labgrid/driver/powerdriver.py]: requests will raise an exception in that case. ;)

Comment (Emantor) [reply labgrid/driver/powerdriver.py]: iPyhton disagrees:
```
In [4]: requests.get("https://httpstat.us/503")  
Out[4]: <Response [503]>

In [5]:                                          
```
According to the [docs](https://requests.kennethreitz.org/en/master/user/quickstart/#response-status-codes) `raise_for_status()` needs to be called on the response object.

Comment (jluebbe) [reply labgrid/driver/powerdriver.py]: Thanks! I've fixed this and updated the PR.

**Pull request merged**